controllers/project_controller_test.go

```diff
@@ -0,0 +1,101 @@
+package controllers
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	mockData "github.com/stakater/jira-service-desk-operator/mock"
+
+	jiraservicedeskv1alpha1 "github.com/stakater/jira-service-desk-operator/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var _ = Describe("ProjectController", func() {
+
+	var name string
+	var key string
+	var projectTypeKey string
+	var projectTemplateKey string
+	var description string
+	var assigneeType string
+	var leadAccountId string
+	var url string
+
+	BeforeEach(func() {
+		name = mockData.CreateProjectInput.Name
+		key = mockData.CreateProjectInput.Key
+		projectTypeKey = mockData.CreateProjectInput.ProjectTypeKey
+		projectTemplateKey = mockData.CreateProjectInput.ProjectTemplateKey
+		description = mockData.CreateProjectInput.Description
+		assigneeType = mockData.CreateProjectInput.AssigneeType
+		leadAccountId = mockData.CreateProjectInput.LeadAccountId
+		url = mockData.CreateProjectInput.URL
```

Comment:
All of this is static data, instead of assigning these values like this create a struct that would encapsulate these values and then retrieve that. Also, this has nothing dynamic so it shouldn't be a part of `beforeEach`